include/control_toolbox/pid.h

```diff
@@ -124,7 +124,7 @@ class Pid
   struct Gains
   {
     // Optional constructor for passing in values
-    Gains(double p, double i, double d, double i_max, double i_min, bool antiwindup)
+    Gains(double p, double i, double d, double i_max, double i_min, bool antiwindup = false)
```

Comment:
This will break ABI, although not API.

You can duplicate the method and add one w/o `bool antiwindup` to keep ABI compatibility.

If it's not a problem breaking the ABI within kinetic releases, then ignore this. @bmagyar ?